src/__experimental__/components/search/search.component.js

```diff
@@ -172,6 +184,7 @@ const Search = ({
         onBlur={onBlur}
         onChange={onChange}
         onKeyDown={onKeyDown}
+        inputRef={assignInput}
```

Comment:
`Textbox` doesn't actually have an `inputRef` prop so it's worth adding it to `Textbox` here as well, it just works how you've got it because `Textbox`'s props are spread down to `Input` (which they shouldn't be)